test/config.sh

```diff
@@ -95,6 +95,12 @@ assert_dns_record() {
 }
 
 start_suite() {
+    for HOST in $HOST1 $HOST2; do
```

Comment:
I ended up putting this in `start_suite` rather than `end_suite` as the latter isn't run in cases of abnormal test case termination, leaving the system in a broken state in which none of the tests case run.